admin_guide/topics/deploying-cluster-auto-scaler.adoc

```diff
@@ -5,11 +5,20 @@
 [id='deploying-cluster-auto-scaler-{context}']
 = Deploying the auto-scaler components on your cluster
 
-Once we have created the ASG and its associated LC we can deploy the
-auto-scaler components onto the cluster.
+After you create the Launch Configuration and Auto Scaling group, you can deploy
+the auto-scaler components onto the cluster.
+
+.Prerequisites
+
+* Install a {product-title} cluster in AWS.
+* Create a primed image.
+* Create a Launch Configuration and Auto Scaling group that reference the primed
+image.
 
 .Procedure
-. Create namespace
+On the same host that you used to deploy the cluster:
```

Comment:
nit; the same host doesn't matter. You could actually run these on commands on the cluster itself which, given the nature of the examples, is what I do when testing.